### PR TITLE
DEVDOCS-5655: [update] b2b-order

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5655]

## What changed?
Updated the order.yml 

## Release notes draft
N/A

## Anything else?

ping {names}

[DEVDOCS-5655]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5655?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/b2b-edition/specs/storefront/storefront/order.yaml [hunk 1]
```diff
@@ -1500,7 +1529,7 @@ paths:
                           - shipments
                           - poNumber
                           - money
-                      - $ref: https://raw.githubusercontent.com/bigcommerce/docs/main/docs/b2b-edition/models/order/order-extra.yaml
+                      - $ref: '../../../models/order/order-extra.yaml'
```
Comment (becomevocal) [inline docs/b2b-edition/specs/storefront/storefront/order.yaml:1532]: Why was this update? I think it's clearer to have the full URL here. 🤔

Comment (bc-traciporter) [reply docs/b2b-edition/specs/storefront/storefront/order.yaml:1532]: The full URL causes errors. The errors appear in visual studio. I had to update all of them.

File: docs/b2b-edition/specs/storefront/storefront/order.yaml [hunk 2]
```diff
@@ -3,8 +3,14 @@ info:
...
   version: '1.0'
   description: Company based order management.
+  contact:
+    name: BigCommerce
+    url: 'https://www.bigcommerce.com'
+    email: support@bigcommerce.com
 servers:
   - url: 'https://api-b2b.bigcommerce.com/api/v2'
+security:
+  - authToken: []
```
Comment (BC-EEspinosa) [inline docs/b2b-edition/specs/storefront/storefront/order.yaml:13]: For all B2B storefront API's the recommended way to authenticate is `Authorization: Bearer {B2B_JWT_TOKEN}`

**Review by BC-EEspinosa — COMMENTED** [review]
I recommend to update the security token for the entire storefront API's.  Bearer token is a standard way to do it.
It wasn't in original documentation due it was adopted broadly for B2B  storefront API's last year

**Comment by bc-traciporter** [comment]
Closing this PR, I moved changes to PR 569.

**Pull request closed without merging**